src/main/java/task/core/TaskManager.java

```diff
@@ -30,19 +35,30 @@
 
     private IJobRecordConnection monitorConnection;
 
+    /**
+     * Constructor for a TaskManager object
```

Comment:
No need to explain this. You may want to mention at the top of the class, this is a singleton bec. of A, B, C reasons.